src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```
@@ -113,8 +113,6 @@ private static void InitializeAccessibilityDelegateAccessibleInterface()
         private static ulong AccessibilityCalculateStatesWrapper(IntPtr self, ulong initialStates)
         {
             View view = GetViewFromRefObject(self);
-            if (view == null)
```

If view is null, then the following code is not accessible?
Or view cannot be null?
```
bitMask = view.AccessibilityCalculateStates().BitMask;
```

---

It cannot be null, see below.

---

Could you please throw exception than? If so, it might be helpful when some unknown error occured in future.

---

Okay, added throwing `ArgumentException` in both `GetViewFromRefObject` and `GetInterfaceFromRefObject`.